scripts/mapps-ci

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Purpose of this method in a comment would be nice.

Reply:
Its not needed anymore, I forgot to close the PR